Documentation/teaching/labs/block_device_drivers.rst

```diff
@@ -776,9 +777,9 @@ segments, and will also update global information stored in an iterator
 internal information (segment vector index, number of bytes left to be
 processed, etc.) .
 
-It can store information in the mapped buffer, or extract information.
+You can store information in the mapped buffer, or extract information.
 
-In the case request queues are used and it is needed to process the requests
+In case request queues are used and you needed to process the requests
```

Comment:
s/needed/need

Reply:
Merged this. @Sergiu121  you can send a patch on top.